core/api.py

```diff
@@ -181,7 +191,7 @@ async def ingest_text(
             operation_type="ingest_text",
             user_id=auth.entity_id,
             tokens_used=len(request.content.split()),  # Approximate token count
-            metadata=request.metadata.model_dump() if request.metadata else None,
+            metadata=request.model_dump() if request else None,
```

Comment:
se shouldn't dump the whole request in metadata. Is this required? metadata is only user specific.